lib/utils.pm

```diff
@@ -1199,7 +1199,7 @@ sub set_hostname {
 
                 # Default timeout (10 seconds) may be too short with qemu NO kvm, so increase to 20s - poo#131366
                 my $nmcli = get_var('QEMU_NO_KVM') ? 'nmcli -w 20' : 'nmcli';
-                assert_script_run "$nmcli device disconnect $dev";
```

Comment:
the line above can be modified, so `nmcli -w 60` unconditionally and treat any further failures as a bug in the network stack.